tools/test/spidermonkey/excludelist.txt

```diff
@@ -10,6 +10,9 @@ non262/extensions/regress-367501-03.js
 non262/extensions/regress-367501-04.js
 
 # Skip, These are dependent on implementation or not standard.
+non262/Array/shift-no-has-trap.js
+non262/Array/pop-no-has-trap.js
+non262/Array/concat-proxy.js
```

Comment:
These skipped spidermonkey TCs are still valid in other major browsers (I tested it on chrome and firefox both).
Would you please check it once again?

Reply:
i fixed the error. The test-cases has been removed from the skip list.